tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Builders/OrganisationControllerBuilder.cs

```diff
@@ -14,16 +17,21 @@ internal sealed class OrganisationControllerBuilder
     {
         private IOrganisationRepository _organisationRepository;
         private ICreateOrganisationService _createOrganisationService;
+        private ClaimsPrincipal ClaimsPrincipal;
```

Comment:
This can be `readonly`.

The name should also include the lovely underscore, so `_claimsPrincipal`.

Reply:
👍